app/models/user.rb

```diff
@@ -1,12 +1,20 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Well done @mryeminthu, good job, your application is working but not as expected, you should add the `: confirmable` attribute, which will be responsible for confirming the email of the user before the account creation phase. At this moment, a person can create an account even having the right email ID. It is also true that you are not going to do the actual email verification, but you can mimick the behaviour by printing the `confirmation` link into the terminal, by the use of which a person will be able to get confirmed their identity, which not only saves the space of our database but with great authenticity.